integration-tests/elasticsearch-rest-client/pom.xml

```diff
@@ -112,6 +122,14 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>11</source>
+          <target>11</target>
+        </configuration>
+      </plugin>
```

Comment:
That's a bit unexpected ... 
I'd expect that the compiler plugin configuration would come from the parent poms. We shouldn't override the versions in a test module... was there a particular reason for adding it?
```suggestion
```

Reply:
I did not do this intentionally.
I think I just used some suggestion of IDE and missed, when pushed commit

Reply:
Inattention. I accepted the IDE's offer, but then overlooked it when reviewing the commit.